docs/orchestration/README.md

```diff
@@ -4,8 +4,8 @@ MinIO is a cloud-native application designed to scale in a sustainable manner in
 
 | Orchestration platforms                                                                            |
 |:---------------------------------------------------------------------------------------------------|
-| [`Kubernetes`](https://docs.min.io/docs/deploy-minio-on-kubernetes)                                |
-| [`Docker Compose`](https://docs.min.io/docs/deploy-minio-on-docker-compose) (only for test setups) |
+| [`Kubernetes`](https://min.io/docs/minio/kubernetes/upstream/index.html#quickstart-for-kubernetes)                                |
+| [`Docker Compose`](https://min.io/docs/minio/container/index.html) (only for test setups) |
```

Comment:
I'd drop 'compose' since we are not attempting to maintain those instructions at this time.

Reply:
Dropped